docs/usage.md

```diff
@@ -9,36 +9,66 @@ The FortiSOAR Global Visibility Solution Pack provides a comprehensive set of to
 
-When a user clicks on a Record Summary Card representing a site, the widget broadcasts an event containing information about the selected site. This event serves as a trigger for other widgets to adjust their displayed data according to the selected site.
+![Global Visibility Scenario](./res/global-visibility-dashboard-edit-01.png)
 
 ### Record Summary Tile
 The Record Summary Tile widget offers a more detailed breakdown of record metrics for all sites combined. It complements the Record Summary Card by providing a compact visualization of metrics such as Total Alerts, Total Indicators, Total Tenants.
 
+![Global Visibility Scenario](./res/global-visibility-dashboard-widget-01.png)
+
+### Record Summary Card
+As described earlier, the Record Summary Card widget provides an overview of record metrics for each site. Additionally, it has the unique ability to broadcast an event when clicked. This event triggers data updates in other widgets.
+
+![Global Visibility Scenario](./res/global-visibility-dashboard-widget-02.png)
+
+#### Broadcast Event
+When a user clicks on a Record Summary Card representing a site, the widget broadcasts an event containing information about the selected site. This event serves as a trigger for other widgets to adjust their displayed data according to the selected site.
 
 ### Funnel Chart
 The Funnel Chart widget visualizes the progression of records through various stages of the workflow. It's an excellent tool for understanding the efficiency of your incident response process.
 
+![Global Visibility Scenario](./res/global-visibility-dashboard-widget-04.png)
+
 #### Event Listening
 
 The Funnel Chart widget listens for the broadcasted event triggered by the Record Summary Card. When the event is received, the Funnel Chart updates to show the funnel for the selected site.
 
 ### Top-X
 The Top-X widget identifies the top records based on specific criteria, such as highest priority or most recent. It helps users focus on critical records and analyze trends across sites.
 
+![Global Visibility Scenario](./res/global-visibility-dashboard-widget-03.png)
+
 #### Event Listening
 
 Similar to the Funnel Chart, the Top-X widget also listens for the broadcasted event from the Record Summary Card. When the event is detected, the Top-X widget adjusts its data to reflect the selected site.
 
-### Image Viewer
-The Image Viewer widget allows users to display images or diagrams that are relevant to the selected site. This visual aid can provide contextual information and enhance the decision-making process.
+### Richtext Content
+The Richtext Content widget allows users to display images or diagrams that are relevant to the selected site. This visual aid can provide contextual information and enhance the decision-making process. User can change an image of their choice by modifying `img src` html tag by editing this widget settings.
```

Comment:
The Richtext Content widget helps display images or diagrams that are relevant to the selected site. This visualization provides contextual information and enhances the decision-making process. You can change to an image of your choice by modifying the `<img src=""/>` html tag in this widget's settings.